integration/monai/examples/spleen_ct_segmentation_real-world/job_he/app/config/config_fed_client.json

```diff
@@ -0,0 +1,61 @@
+{
+  "format_version": 2,
+
+  "executors": [
+    {
+      "tasks": [
+        "train", "submit_model", "validate"
+      ],
+      "executor": {
+        "id": "executor",
+        "path": "monai_nvflare.client_algo_executor.ClientAlgoExecutor",
+        "args": {
+          "client_algo_id": "client_algo",
+          "key_metric": "val_mean_dice"
+        }
+      }
+    }
+  ],
+
+
+  "task_result_filters": [
+    {
+      "tasks": ["train"],
+      "filters":[
+          {
+          "name": "HEModelEncryptor",
+          "args": {
+            "aggregation_weights": {
+              "site1":  1.0,
+              "site2":  1.0
+            }
+          }
+        }
+      ]
+    }
+  ],
+
+  "task_data_filters": [
+    {
+      "tasks": ["train", "validate"],
+      "filters":[
+          {
+          "name": "HEModelDecryptor",
+          "args": {
+          }
+        }
+      ]
+    }
+  ],
+
+  "components": [
+    {
+      "id": "client_algo",
+      "path": "monai.fl.client.MonaiAlgo",
+      "args": {
+        "bundle_root": "config/spleen_ct_segmentation",
```

Comment:
Does this need to change to "spleen_ct_segmentation_real-world"?

Reply:
No, bundle_root is relative to the app_root.